site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -0,0 +1,364 @@
+author: sfc-gh-drichert
+id: servicenow_to_snowflake_connector
+summary: Step-by-step to set up Servicenow connector
+categories: Connectors
+environments: web
+status: Private 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Connectors, Data Engineering, Servicenow 
+
+# Snowflake Connector for ServiceNow Installation
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+Use this quickstart lab to configure and understand the Snowflake Connector for ServiceNow using the Snowsight wizard, select some tables, ingest data, and run an example query. This quickstart is not meant to be exhaustive. Please check the [Snowflake Conector for ServiceNow documentation](https://https://other-docs.snowflake.com/en/connectors/servicenow/servicenow-index.html) for full functionality and limitations.
+
+![now](assets/first.png)
+
+> aside positive
+> Note: This quickstart assumes you do not have a ServiceNow account, so it guides you through the steps of creating a developer account. Of course, if you do have a Servicenow account, please feel free to try it out, with the caveat that, at the time of writing, the connector is in public preview and should not be used for production. 
+### Prerequisites
+- ServiceNow account with administrator's rights.
+- ORGADMIN rights to Accept the Terms of Service in the Snowflake Marketplace.
+- ACCOUNTADMIN rights on the Snowflake account where you will install the connector.
+
+### What You’ll Learn 
+- How to set up the Snowflake Connector for ServiceNow.
+- How to ingest ServiceNow data into Snowflake
+- How to stop the connector to avoid unnecessary costs in a development environment.
+### What You’ll Need 
+- A [Snowflake](https://snowflake.com/) Account 
+- A [ServiceNow](https://developer.servicenow.com/dev.do/) developer account
+### What You’ll Build 
+A ServiceNow to Snowflake ingestion data flow.
+
+<!-- ------------------------ -->
+## Set up the ServiceNow Developer Instance
+Duration: 5
+
+If you do not want to test this connector on your ServiceNow account, no problem, this step explains how to set up a developer instance! 
+
+1. Go to the [ServiceNow developer website](https://developer.servicenow.com), and create a developer user.
+
+1. Log on to the developer website with your newly created user and select **Create an Instance**. 
+1. Choose an instance type. You receive an email with your instance URL, and admin user and password. 
+
+Deployment is usually pretty quick, around five minutes. But, while you wait let's go to the next step and configure Snowflake!
+## Create and set up the Snowflake Account
+
+### Create the Snowflake Account
+If you do not have a Snowflake account, no problem, you can get a free trial  at [snowflake.com](https://www.snowflake.com/en/).Select **Start for Free** and follow the instructions. You will probably want to deploy in the same region as the ServiceNow deployment, but this is not a requirement for the lab. 
```

Comment:
I don't think we have any kind of recommendations regarding the region of Snowflake account in regard to SN instance. Will confirm with the team but if we do not, then I think we should remove it.